values.yaml

```diff
@@ -300,6 +300,12 @@ core:
     repository: goharbor/harbor-core
     tag: dev
   replicas: 1
+  livenessProbe:
```

Comment:
@bpehling I think we can just make the `initialDelaySeconds` of `livenessProbe` configurable to fix the upgrading issue and with less configuration items for users.

Please add comment for the item to explain what is it for and document it in the `README.md`

Reply:
And larger the delay time, maybe 300 is a reasonable value?

Reply:
I think 300 is reasonable, fixed

Reply:
Thanks